sec-keys/openpgp-keys-canonical/openpgp-keys-canonical-20190401.ebuild

```
@@ -0,0 +1,19 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+DESCRIPTION="OpenPGP keys used to sign LXD-related packages"
+HOMEPAGE="https://ubuntu.com/lxd"
+SRC_URI="https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xed1ca1e7a6f80e22e5cb2da84ace106615754614 -> 15754614.asc"
+
+LICENSE="public-domain"
+SLOT="0/stable"
```

```suggestion
SLOT="0"
```

We don't have to differentiate this since we control the dependency on lxd package.

---

Fixed